WordPress/Classes/Categories/NSString+Helpers.m

```diff
@@ -188,4 +188,13 @@ - (NSArray *)tokenize
     return tokens;
 }
 
+- (BOOL)isWordPressComURL
+{
+    NSRegularExpression *protocol = [NSRegularExpression regularExpressionWithPattern:@"wordpress\\.com/?$" options:NSRegularExpressionCaseInsensitive error:nil];
```

Comment:
I'd think using NSURLComponents would be much safer than Regex. I hate Regex. :smile: